bin/resources/GameIndex.yaml

```diff
@@ -5217,11 +5213,11 @@ SCPS-15006:
   name: "Popolocrois Story 3 [Limited Edition]"
   region: "NTSC-J"
 SCPS-15007:
-  name: "Blood - The Last Vampire - Vol.1 Joukan"
+  name: "Blood - The Last Vampire - Joukan [Volume 1]"
```

Comment:
In this case, the game is divided in 2 volumes, the first being Joukan (Vol 1) and the last being Gekan (Vol 2). I've tried to keep the original title without translating it but it seems it can't be helped. Reverted.